package.json

```diff
@@ -24,7 +24,7 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
-    "clean": "rm -rf ./packages/client/__generated__ ./queryMap.json ./packages/server/graphql/public/schema.graphql ./packages/server/graphql/private/schema.graphql ./dev ./dist ./build",
+    "clean": "mv .env .env-backup && git add .env-backup && git clean -fdx && git reset .env-backup && mv .env-backup .env",
```

Comment:
-1 I did not test, but it seems there is an `--exclude` argument for exactly this purpose: https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--eltpatterngt

Reply:
oh, that's great! thank you.

Reply:
fixed!